Add version to docs

KrishnaIyer opened the pull request:
<!--
Thanks for submitting a pull request. Please fill the template below,
otherwise we will not be able to process this pull request.
-->

#### Summary
<!--
A short summary, referencing related issues:
Closes #0000, References #0000, etc.
-->

Add version to docs footer. 
References https://github.com/TheThingsIndustries/lorawan-stack/issues/1832


#### Changes
<!-- What are the changes made in this pull request? -->

- Add version to docs footer
- Remove Github commit info and edit button.

#### Notes for Reviewers
<!--
NOTE: This section is optional.

Motivate briefly why it is implemented this way, if that deviates from the
implementation proposal in the referenced issues.
- How should your reviewers approach this pull request?
- @mention reviewers with special requests or questions for them
-->

 The **Edit on Github** was removed since it directly jumps to the Github editor. It's preferable that people clone the stack, add changes, test locally and create a PR instead.
Please let me know if this needs to be reinstated.

#### Checklist
<!-- Make sure that this pull request is complete. -->

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [ ] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

htdvisser commented on doc/themes/the-things-stack/layouts/partials/file-info.html:
```diff
@@ -1,18 +1,5 @@
-  <br>
-  {{ if .GitInfo -}}
-  {{- if .Site.Params.github_repository -}}
-    <a href="{{ .Site.Params.github_repository }}/commit/{{ .GitInfo.Hash }}">
```
Why remove this? It can be very useful to see what changed exactly.

htdvisser commented on doc/themes/the-things-stack/layouts/partials/file-info.html:
```diff
@@ -1,18 +1,5 @@
-  {{- else -}}
-    {{ .GitInfo.Subject }}
-  {{- end }}{{ end }}
+<p class="has-text-light is-size-2">
```
I think we should put the version somewhere in the top of the page (where people can see it), and not in the footer (where people don't see it unless they go looking for it)

KrishnaIyer commented:
@htdvisser: I've reverted my commits and added the version next to the logo. Here's how it looks;

**Full Width**
<img width="1436" alt="Screenshot 2020-03-02 at 19 02 26" src="https://user-images.githubusercontent.com/13186113/75703825-8ac31500-5cb8-11ea-94b9-bea34e265259.png">

**Mobile View Port**
<img width="410" alt="Screenshot 2020-03-02 at 19 02 36" src="https://user-images.githubusercontent.com/13186113/75703831-8bf44200-5cb8-11ea-9333-155842df9630.png">

Let me know what you think.
